src/reactviews/common/locConstants.ts

```diff
@@ -316,6 +316,12 @@ export class LocConstants {
             copy: l10n.t("Copy"),
             copyWithHeaders: l10n.t("Copy with Headers"),
             copyHeaders: l10n.t("Copy Headers"),
+            null: l10n.t("NULL"),
+            blankString: l10n.t("Blanks"),
+            apply: l10n.t("Apply"),
+            clear: l10n.t("Clear"),
+            search: l10n.t("Search..."),
+            selected: l10n.t("0 Selected"),
```

Comment:
If this is only shown when none are selected, then the var name should be `noneSelected` or `zeroSelected` instead of `selected`